### ISSUE TITLE
Missing documentation on how to add authenticated users

### ISSUE THREAD

**Issue by junjun-zhang** [nlesc-nano/ceiba#30]
When running the `ceibacli login` command using the GitHub PAT generated from my account, I got an error message: `User `junjun-zhang` doesn't have permissions to access the service`. Checked the ceiba code and noticed that there is a Mongo DB collection `authenticated_users`, however, there seems to be no documentation on how to add users to the collection.

**Comment by junjun-zhang** [comment]
Thanks @felipeZ for the quick response, however, the problem persists even I edited the `users.txt` file.

Had a look at the docker compose file, based on this line:
`entrypoint: ceiba -m 172.17.0.2 -u admin -p ${MONGO_PASSWORD} -f users.txt`

The `users.txt` file has to be within the docker contain, not the one from the host. Basically it would require the docker image to be rebuilt in order to include the updated `users.txt` included in the container. Alternatively, a volume can be mount that include this file.

**Comment by felipeZ** [comment]
@junjun-zhang You are right about it. I just forgot about the volume, actually, I had opened issue #16  to acknowledge that issue but I forgot about it.
